notebook/static/base/js/security.js

```diff
@@ -31,6 +31,10 @@ define([
                 }
             }
         }
+        // TODO(julienr): This is a ugly hack to work around the fact that
+        // by default caja doesn't seem to like base64 src for inline markdown
+        // images. Not sure if we should do that here
+        ATTRIBS['img::src'] = 0;
```

Comment:
The good solution would be to only allow `data` uris (and disallow `javascript:` uri). Unfortunately, it seems that caja has hardcoded allowed uri schemes (and `data` is not included):

https://github.com/minrk/google-caja-bower/blob/master/html-sanitizer.js#L813

Should I just monkey patch `caja.html.ALLOWED_URI_SCHEMES` to allow data uris ?

There is an open issue on caja on this subject, but it doesn't seem very alive :
https://github.com/google/caja/issues/1558